catalog/core/catalog-core-directorymonitor/src/main/java/org/codice/ddf/catalog/content/monitor/FileSystemPersistenceProvider.java

```diff
@@ -95,7 +96,8 @@ String getPersistencePath() {
      * @return
      */
     String getMapStorePath() {
-        return getPersistencePath() + mapName + File.separator;
```

Comment:
There doesn't seem to be any reason for not using `AbsolutePathResolver`. I will replace this `getPersistencePath()` with that implementation for consistency.